### PR TITLE
fix: golangci-lint errors and warnings

### PR DESCRIPTION
**Pull request by bhoopesh369**
Closes https://github.com/opiproject/sztp/issues/390

### CODE REVIEW

**Comment by glimchb** [comment]
Thanks @bhoopesh369 for you first contribution to OPI, we'll review this PR asap

File: .golangci.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ run:
 # output configuration options
 output:
   # colored-line-number|line-number|json|tab|checkstyle|code-climate, default is "colored-line-number"
-  format: colored-line-number
+  formats: colored-line-number
```
Comment (glimchb) [inline .golangci.yml:10]: typo? `s` seems wrong here
look at https://golangci-lint.run/usage/configuration/

**Review by glimchb — CHANGES_REQUESTED** [review]
great work! appreciate your contribution
small typo to fix (i think) correct me if I'm wrong

File: .golangci.yml (hunk 1 above)
Comment (bhoopesh369) [reply .golangci.yml:10]: image

output.format was used in older version golangci-lint
it was changed to output.formats in the later versions

Comment (bhoopesh369) [reply .golangci.yml:10]: format was used in older versions
it is still supported but yea gives a warning

i actually made a mistake here it shoud have been : 
 ```
 formats:
       - format: colored-line-number
         path: stdout
         
```

Comment (bhoopesh369) [reply .golangci.yml:10]: i'll change it and push

Comment (glimchb) [reply .golangci.yml:10]: thanks

**Review by glimchb — APPROVED** [review]
lgtm, thanks for fixing

**Pull request merged**